### PR TITLE
Webroot method

### PR DESCRIPTION
**Pull request by dzotic91-ukr**

### CODE REVIEW

File: scripts/ssl-manager.js [hunk 1]
```diff
@@ -468,6 +473,51 @@ function SSLManager(config) {
         return me.getFileUrl("scripts/" + scriptName);
     };
 
+    me.initCustomConfigs = function() {
+        var CUSTOM_CONFIG = "/var/lib/jelastic/keys/letsencrypt/settings-custom",
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: hardcode. What about custom docker templates?

File: scripts/ssl-manager.js [hunk 2]
```diff
@@ -468,6 +473,51 @@ function SSLManager(config) {
...
+
+        body = resp.body.replace(/\n$/, '');
+        configs = body.split('\n');
+
+        for (var i = 0, n = configs.length; i < n; i++) {
+            tmp = configs[i].split('=');
+            config[tmp[0]] = convertToBoolean(tmp[1].replace(/\"/g, ""));
+        }
+
+        jelastic.marketplace.console.WriteLog("config.webroot22 -> " + config.webroot);
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: remove

File: scripts/ssl-manager.js [hunk 3]
```diff
@@ -468,6 +473,51 @@ function SSLManager(config) {
...
+        jelastic.marketplace.console.WriteLog("config.webroot22 ? nodeManager.getMasterIdByLayer(CP) : config.nodeId -> " + config.webroot ? nodeManager.getMasterIdByLayer(CP) : config.nodeId);
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: remove

File: scripts/ssl-manager.js [hunk 4]
```diff
@@ -499,8 +549,11 @@ function SSLManager(config) {
                 config.nodeId = node.id;
                 config.nodeIp = node.address;
 
-                nodeManager.setNodeId(config.nodeId);
-                nodeManager.setNodeIp(config.nodeIp);
+                jelastic.marketplace.console.WriteLog("config.webroot -> " + config.webroot);
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: remove

File: scripts/ssl-manager.js [hunk 5]
```diff
@@ -499,8 +549,11 @@ function SSLManager(config) {
...
+                jelastic.marketplace.console.WriteLog("config.webroot ? nodeManager.getMasterIdByLayer(CP) : config.nodeId -> " + config.webroot ? nodeManager.getMasterIdByLayer(CP) : config.nodeId);
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: remove

File: scripts/ssl-manager.js [hunk 6]
```diff
@@ -1056,6 +1110,31 @@ function SSLManager(config) {
         me.isBalancerLayer = function (group) {
             return !!(group == LB || group == BL);
         };
+        
+        me.getEntryNodeIps = function() {
+            jelastic.marketplace.console.WriteLog("config.webroot22 before resp -> ");
```
Comment (SlavaKatiukha) [inline scripts/ssl-manager.js]: remove

File: scripts/ssl-manager.js (hunk 4 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: +

File: scripts/ssl-manager.js (hunk 5 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: +

File: scripts/ssl-manager.js (hunk 6 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: done

File: scripts/ssl-manager.js (hunk 3 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: done

File: scripts/ssl-manager.js (hunk 2 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: done

File: scripts/ssl-manager.js (hunk 1 above)
Comment (dzotic91-ukr) [reply scripts/ssl-manager.js]: fixed

**Pull request closed without merging**